pkg/csi/service/vanilla/controller.go

```diff
@@ -1031,12 +1021,12 @@ func (c *controller) ControllerGetCapabilities(ctx context.Context, req *csi.Con
 
 	var controllerCaps []csi.ControllerServiceCapability_RPC_Type
 
-	vsan67u3Release, err := isVsan67u3Release(ctx, c)
+	isSupported, err := c.manager.VcenterManager.IsExtendVolumeSupported(ctx, c.manager.VcenterConfig.Host)
 	if err != nil {
-		log.Error("failed to get vcenter version to help identify controller service capabilities")
+		log.Error("failed to verify if extend volume is supported or not. Error:%+v", err)
 		return nil, status.Error(codes.FailedPrecondition, err.Error())
 	}
-	if vsan67u3Release {
+	if !isSupported {
```

Comment:
Just because IsExtendVolumeSupported returns false, means we are applying VSAN67u3ControllerServiceCapability, this sounds weird to me. 

Can we refactor this further and generate capabilities depending on the result and return, rather than using fixed VSAN67u3ControllerServiceCapability and VSAN7ControllerServiceCapability?